### PR TITLE
[SMALLFIX] Clean up in the test module

### PR DESCRIPTION
**Pull request by HorizonNet**
Minor fixes in the **test** module.

*ChgrpCommandTest*: Added documentation.
*ChmodCommandTest*: Added documentation.
*ChownCommandTest*: Added documentation.
*LocalMiniDFSCluster*: Fixed documentation.

### CODE REVIEW

File: tests/src/test/java/alluxio/shell/command/ChgrpCommandTest.java [hunk 1]
```diff
@@ -41,7 +41,8 @@ public void chgrpTest() throws IOException, AlluxioException {
   /**
    * Tests -R option for chgrp recursively.
    *
-   * @throws Exception
+   * @throws IOException if creating the file or getting the status of the filesystem fails
+   * @throws AlluxioException if getting the status of the filesystem fails
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/shell/command/ChgrpCommandTest.java]: We're trying to steer away from javadoc-ing exceptions in tests, or declaring explicit exceptions. The rationale is that unlike production code, tests are never supposed to throw an exception, and no other code calls test methods.

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comment. Will keep that in mind. Merged the latest changes into this one.

**Comment by HorizonNet** [comment]
Jenkins, test this please.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**